.github/workflows/tests.yml

```
@@ -0,0 +1,86 @@
+        PYTHON_VERSION: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{matrix.PYTHON_VERSION}}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{matrix.PYTHON_VERSION}}
+
+      - name: Get full Python version
+        id: full-python-version
+        run: echo ::set-output name=version::$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)